blockchain/utxobackend.go

```diff
@@ -21,6 +21,10 @@ type UtxoBackend interface {
 	// When there is no entry for the provided output, nil will be returned for
 	// both the entry and the error.
 	FetchEntry(outpoint wire.OutPoint) (*UtxoEntry, error)
+
+	// PutUtxos atomically updates the UTXO set with the entries from the provided
+	// map along with the current state.
+	PutUtxos(utxos map[wire.OutPoint]*UtxoEntry, state *utxoSetState) error
```

Comment:
Since this is an interface, the returned type should be exported as it would otherwise be unusable outside of the package.

Reply:
Updated.